### PR TITLE
Fix bug 1234501: Make profile toggles work

### PR DESCRIPTION
**Pull request by mathjazz**
@jotes @sudheesh001 r?

### CODE REVIEW

File: pontoon/base/static/js/main.js [hunk 1]
```diff
@@ -597,7 +597,7 @@ $(function() {
     Pontoon.startLoader();
 
     $.ajax({
-      url: '/api/v1/user/' + Pontoon.user.email + '/',
+      url: '/api/v1/user/' + $('#server').data('email') + '/',
```
Comment (jotes) [inline pontoon/base/static/js/main.js]: Maybe we should move initialization of user data before this line to keep one source of data for the user profile in javascript?

Comment (mathjazz) [inline pontoon/base/static/js/main.js]: This method is also used by user.js, where we don't have Pontoon.user.

**Comment by sudheesh001** [comment]
@mathjazz r- ; The Ajax query results in `OOPS SOMETHING WENT WRONG` with the Network information throwing a 404 in request for the `http://localhost:8000/api/v1/user/undefined/`

File: pontoon/base/static/js/main.js (hunk 1 above)
Comment (jotes) [inline pontoon/base/static/js/main.js]: Sure, no problem.

**Comment by jotes** [comment]
@sudheesh001 yup it probably won't work for anonymous users.

**Comment by sudheesh001** [comment]
@jotes I am logged in :+1: In one browser as admin and in another as a regular user.

**Comment by mathjazz** [comment]
That's weird, what does `$('#server').data()` return for you? Did you merge OK?

**Comment by sudheesh001** [comment]
@mathjazz It returns the csrf `Object {csrf: "cOoPkHfO0JRKU1D63bTUjrr1tZca4xku"}`

**Comment by mathjazz** [comment]
It should also return email, you either didn't merge correctly or you're not authenticated;
https://github.com/mozilla/pontoon/pull/311/files#diff-5993865859aacd25a269caf6f0218b3cR13

**Comment by sudheesh001** [comment]
@mathjazz I guess so, seems to be working fine now, I was able to see the
```html
<div id=​"server" class=​"hidden" data-csrf=​"vNiPMATEweL0iuGZlQvao0l4mIfCwlZm" data-email=​"root@root.com">​
​</div>​
```
but the `$('#server').data()` ended up only with `csrf`, It was probably the cache, seems to be working fine now. r+ :smile: Sorry for the confusion

**Comment by jotes** [comment]
post r+

**Pull request merged**